configuration/configuration_test.go

```diff
@@ -54,145 +58,301 @@ func TestNewConfigurationDataFail(t *testing.T) {
 
 func TestGetKeycloakEndpointToken(t *testing.T) {
 	t.Parallel()
+
 	cd := getConfigurationDataHandler()
+	require.NotNil(t, cd)
+
+	viperValue := cd.GetKeycloakEndpointToken()
 
-	t.Log(cd.GetKeycloakEndpointToken())
-	assert.NotNil(t, cd.GetKeycloakEndpointToken())
+	testConfigFileMap, err := getConfigFileMap("")
+	require.Nil(t, err)
+	actualValue := testConfigFileMap.KeycloakEndpointToken
+
+	assert.Equal(t, actualValue, viperValue)
```

Comment:
Yeah, I actually named it wrong, let me rename it :)
`viperValue` is the actual :|